subsystem/utils_test.go

```diff
@@ -27,7 +27,7 @@ func strToUUID(s string) *strfmt.UUID {
 }
 
 func registerHost(clusterID strfmt.UUID) *models.Host {
-	host, err := bmclient.Installer.RegisterHost(context.Background(), &installer.RegisterHostParams{
+	host, err := agentBMClient.Installer.RegisterHost(context.Background(), &installer.RegisterHostParams{
```

Comment:
Need to add a few more calls to this test just to make sure agent can call these APIs
GET  /clusters/{cluster_id}/downloads/files
POST /clusters/{cluster_id}/uploads/ingress-cert

Reply:
Done.
1. GET `DownloadClusterFiles` will now get called with **agent auth** in subsystem test `[only_k8s]install download_config_files`
2. POST `UploadClusterIngressCert` using **agent auth** is already being used in`cluster_test.go`([here for example](https://github.com/openshift/assisted-service/blob/master/subsystem/cluster_test.go#L732-L742))

Reply:
Implemented in [PR-70](https://github.com/openshift/assisted-service/pull/70/files) `subsystem/cluster_test.go `